apps/shipments/signals.py

```diff
@@ -66,7 +66,15 @@ def shipment_post_save(sender, **kwargs):
                                     funding_type=Shipment.FUNDING_TYPE,
                                     contracted_amount=Shipment.SHIPMENT_AMOUNT)
 
-        Shipment.objects.filter(id=instance.id).update(vault_id=vault_id, vault_uri=vault_uri)
+        instance.vault_id = vault_id
+        instance.vault_uri = vault_uri
+        instance.save()
```

Comment:
We do not want to call instance.save() inside the Shipment post save listener. This will trigger an update on the shipment, which will trigger the shipment update post-save flow, which adds an unnessecary set_vault_hash tx among other things. This line: `Shipment.objects.filter(id=instance.id).update(vault_id=vault_id, vault_uri=vault_uri)` was very intentional.

Reply:
In last review with @mlclay, we decided to have an instance of `HistoricalShipment` with a `null` user for the `vault_id` and `vault_uri` set up on the shipment. That's why I needed to change that line.